src/SIM/SIMoutput.C

```diff
@@ -1801,3 +1821,18 @@ bool SIMoutput::serialize (std::map<std::string,std::string>&) const
             <<"     Restart not supported for "<< this->getName() << std::endl;
   return false;
 }
+
+
+bool SIMoutput::writeAddFuncs(int iStep, int& nBlock, int idBlock, double time)
+{
+  if (myAddScalars.empty())
+    return true;
+
+  for (const auto& func : myAddScalars) {
+    if (!this->writeGlvF(*func.second, func.first.c_str(), iStep,
+                         nBlock, idBlock, time))
+        return false;
+  }
```

Comment:
Braces are not needed since just single-statement (cosmetic)

Reply:
ok